ci/test_wheel.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+WHEELHOUSE="${PWD}/dist/"
+RAPIDS_PY_WHEEL_NAME="kvikio_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 python "${WHEELHOUSE}"
+
+python -m pip install "kvikio-${RAPIDS_PY_CUDA_SUFFIX}[test]>=0.0.0a0" --find-links "${WHEELHOUSE}"
```

Comment:
```suggestion
python -m pip install "$(echo ${WHEELHOUSE}/kvikio_${RAPIDS_PY_CUDA_SUFFIX}*.whl)[test]"
```

Based on https://github.com/rapidsai/build-planning/issues/69, could you please install the wheel file directly instead of using `--find-links`? That'll improve our changes of CI catching issues.